app/ante/fee.go

```diff
@@ -238,6 +238,12 @@ func getValidatorFromOracleMsg(msg sdk.Msg) (sdk.ValAddress, error) {
 			return nil, err
 		}
 		return val, nil
+	case *oracletypes.MsgFeederDelegationConsent:
```

Comment:
Why don't we squash all three cases into one since each case is doing exactly the same thing?

Reply:
Updated, does it look better though?